release/nightly_tests/dataset/benchmark.py

```diff
@@ -50,32 +43,82 @@ def __init__(self, name):
         self.result = {}
         print(f"Running benchmark: {name}")
 
-    def run(self, name: str, fn: Callable[..., Dataset], **fn_run_args):
+    def run_materialize_ds(
+        self,
+        name: str,
+        fn: Callable[..., Dataset],
+        *fn_args,
+        **fn_run_args,
```

Comment:
The difference between `fn_args` and `fn_run_args` isn't clear to me based on the names.
```suggestion
        **fn_kwargs,
```